### PR TITLE
[RUM] Add CORB warning info in troubleshooting

### PR DESCRIPTION
**Pull request by amortemousque**
### What does this PR do?
Add CORB warning info in the troubleshooting section

### Motivation
Better troubleshooting

### Preview

https://docs-staging.datadoghq.com/aymeric/rum-browser-add-corb-warning-info-in-troubleshooting/real_user_monitoring/browser/troubleshooting

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/troubleshooting.md [hunk 1]
```diff
@@ -100,6 +100,13 @@ If an event or a request goes beyond any of the following limitations, it is rej
 | Maximum event size                       | 256 KB       |
 | Maximum intake payload size              | 5 MB         |
 
+## Cross origin read blocking warning
+
+When starting session replay recording with `startSessionReplayRecording()` on Chromium based browser, a CORB warning is printed in the console:
```
Comment (BenoitZugmeyer) [inline content/en/real_user_monitoring/browser/troubleshooting.md:105]: **🔨 warning:** ‏This is not related to `startSessionReplayRecording`. The warning is shown using RUM or Logs alone. (The warning is shown because the intake V2 returns a non-empty JSON object... previously it was used mainly for Session Replay, but now it is used for all intakes)

File: content/en/real_user_monitoring/browser/troubleshooting.md [hunk 2]
```diff
@@ -100,6 +100,13 @@ If an event or a request goes beyond any of the following limitations, it is rej
...
+{{< img src="real_user_monitoring/browser/troubleshooting/corb_warning.png" alt="CORB warning">}}
```
Comment (BenoitZugmeyer) [inline content/en/real_user_monitoring/browser/troubleshooting.md:106]: **💬 suggestion:** ‏I would write the error message in full text so it is searchable

File: content/en/real_user_monitoring/browser/troubleshooting.md (hunk 1 above)
Comment (amortemousque) [reply content/en/real_user_monitoring/browser/troubleshooting.md:105]: Okay, make sens. I'll update the doc

File: content/en/real_user_monitoring/browser/troubleshooting.md (hunk 2 above)
Comment (amortemousque) [reply content/en/real_user_monitoring/browser/troubleshooting.md]: Right, good idea

**Review by bcaudan — APPROVED** [review]

**Review by BenoitZugmeyer — APPROVED** [review]

**Review by hfaivre — APPROVED** [review]
:+1: LGTM

**Review by apigirl — APPROVED** [review]
